cms_status_message/static/src/xml/status_message.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8" ?>
+
+<templates id="qweb_templates" xml:space="preserve">
```

Comment:
Here I'm forced to duplicate the templates as I don't know an easy way to reuse original ir.ui.view templates. I cannot find any docs. I could try this approach tough: https://github.com/odoo/odoo/blob/9.0/addons/website/static/src/js/website.theme.js#L21